e2e/test/scenarios/embedding/sdk-iframe-embedding-setup/select-embed-experience.cy.spec.ts

```diff
@@ -0,0 +1,53 @@
+const { H } = cy;
+
+describe("scenarios > embedding > sdk iframe embed setup > select embed experience", () => {
+  beforeEach(() => {
+    H.restore();
+    cy.signInAsAdmin();
+    H.activateToken("bleeding-edge");
+
+    cy.intercept("GET", "/api/dashboard/**").as("dashboard");
+  });
+
+  it("shows dashboard experience by default", () => {
+    cy.visit("/embed/new");
+    cy.wait("@dashboard");
+
+    const iframe = H.getIframeBody();
+    iframe.within(() => {
```

Comment:
This could lead to human-error. Please refer to this PR https://github.com/metabase/metabase/pull/60152.

tl;dr don't assign values in cypress.

Reply:
Fixed in https://github.com/metabase/metabase/pull/59874/commits/1a6491dc838bd17dbd4a860d773fb5327d1e732e, thanks!